docs/src/content/hardhat-runner/docs/guides/deploying.md

```diff
@@ -1,24 +1,18 @@
 # Deploying your contracts
 
-:::tip
-
-Try [Hardhat Ignition](/ignition) for your deployments! Our new declarative system for deploying smart contracts without getting caught up in execution details.
-
-:::
-
-When it comes to deploying, you can use a deployment system like [Hardhat Ignition](/ignition) or you can deploy your smart contracts using scripts. You can deploy the `Lock` contract from the sample project with a deployment script like this:
+To deploy your contracts, you can use [Hardhat Ignition](/ignition), our declarative deployment system. You can deploy the `Lock` contract from the sample project by specifying a deployment with an Ignition Module file like this:
```

Comment:
it's not specified here where the ignition module goes. It's implied later, but I think being explicit could help those who are not familiar with hardhat / ignition. Something like:

To deploy your contracts, you can use [Hardhat Ignition](/ignition), our declarative deployment system. To deploy the `Lock` contract from the sample project, you need to specify a deployment with an Ignition Module file. Create this file at `./ignition/modules/Lock.ts` within your Hardhat project structure. The Ignition Module file should look like this:

{code block}